### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by yruslan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,20 +59,58 @@ DataWorks Summit 2019 (General Cobrix workflow for hierarchical databases): http
...
 
-#### Linking
+## Requirements
 
-[![Maven Central](https://maven-badges.herokuapp.com/maven-central/za.co.absa.cobrix/spark-cobol/badge.svg)](https://maven-badges.herokuapp.com/maven-central/za.co.absa.cobrix/spark-cobol)
+| spark-cobol | Spark         |
+| ----------- | ------------- |
+| 0.x         | 2.2.x+        |
+| 1.x         | 2.2.x+        |
+| 2.x+        | 2.4.3+        |
```
Comment (Zejnilovic) [inline README.md]: I would just go `2.2+` or `2+`. The Xs at the end seem needless.

File: README.md [hunk 2]
```diff
@@ -189,25 +227,27 @@ to decode various binary formats.
...
 ```
-$ spark-shell --master yarn --deploy-mode client --driver-cores 4 --driver-memory 4G --jars spark-cobol-1.1.2.jar,cobol-parser-1.1.2.jar,scodec-core_2.11-1.10.3.jar,scodec-bits_2.11-1.1.4.jar
+$ spark-shell --packages za.co.absa.cobrix:spark-cobol_2.11:2.0.0
+or 
+$ spark-shell --master yarn --deploy-mode client --driver-cores 4 --driver-memory 4G --jars spark-cobol_2.11-2.0.0.jar,cobol-parser_2.11-2.0.0.jar,scodec-core_2.11-1.10.3.jar,scodec-bits_2.11-1.1.4.jar
 
 Setting default log level to "WARN".
 To adjust logging level use sc.setLogLevel(newLevel). For SparkR, use setLogLevel(newLevel).
-18/09/14 14:05:45 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
+19/12/10 10:01:15 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
```
Comment (Zejnilovic) [inline README.md]: I would just remove this line

File: README.md [hunk 3]
```diff
@@ -1150,6 +1190,9 @@ For multisegment variable lengths tests:
 ![](performance/images/exp3_multiseg_wide_records_throughput.svg) ![](performance/images/exp3_multiseg_wide_mb_throughput.svg)
 
 ## Changelog
+- #### 2.0.0 released 11 December 2019.
+  - Added cross-compilation for Scala `2.11` and `2.12` via `sbt` build (Thanks Georgi Chochov).
```
Comment (Zejnilovic) [inline README.md]: Is there a reason to not use @mention `@GeorgiChochov`?

File: examples/spark-cobol-app/pom.xml [hunk 4]
```diff
@@ -30,9 +30,9 @@
 		<scala.version>2.11.12</scala.version>
 		<scala.compat.version>2.11</scala.compat.version>	
 		<scalatest.version>3.0.0</scalatest.version>
-		<spark.version>2.3.0</spark.version>
+		<spark.version>2.4.4</spark.version>
 		<specs.version>2.4.16</specs.version>
-        <spark.cobol.version>1.1.2</spark.cobol.version>
+        <spark.cobol.version>2.0.0</spark.cobol.version>
```
Comment (Zejnilovic) [inline examples/spark-cobol-app/pom.xml]: there seems to be an error in indentation

File: README.md [hunk 5]
```diff
@@ -59,20 +59,58 @@ DataWorks Summit 2019 (General Cobrix workflow for hierarchical databases): http
...
+## Requirements
 
-[![Maven Central](https://maven-badges.herokuapp.com/maven-central/za.co.absa.cobrix/spark-cobol/badge.svg)](https://maven-badges.herokuapp.com/maven-central/za.co.absa.cobrix/spark-cobol)
+| spark-cobol | Spark         |
+| ----------- | ------------- |
+| 0.x         | 2.2.x+        |
+| 1.x         | 2.2.x+        |
+| 2.x+        | 2.4.3+        |
+
+## Linking
```
Comment (Zejnilovic) [inline README.md]: I am not sure I understand the verb `Linking` in this part of a README

File: examples/spark-cobol-app/build.sbt [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+libraryDependencies ++= Seq(
+  "za.co.absa.cobrix" %% "spark-cobol"     % sparkCobolVersion,
+  "org.scalatest"     %% "scalatest"       % scalatestVersion   % Test,
+  "org.apache.spark"  %% "spark-core"      % sparkVersion       % Provided,
+  "org.apache.spark"  %% "spark-sql"       % sparkVersion       % Provided,
+  "org.apache.spark"  %% "spark-streaming" % sparkVersion       % Provided,
+  "org.scala-lang"    %  "scala-library"   % scalaVersion.value % Provided
+)
+
```
Comment (yruslan) [inline examples/spark-cobol-app/build.sbt]: Use assembly to create a fat jar

File: build.sbt [hunk 7]
```diff
@@ -65,6 +65,4 @@ ThisBuild / coverageExcludedFiles := ".*Example.*;Test.*"
 
 // release settings
 releaseCrossBuild := true
-addCommandAlias("releaseMajor", ";set releaseVersionBump := sbtrelease.Version.Bump.Major; release with-defaults")
-addCommandAlias("releaseMinor", ";set releaseVersionBump := sbtrelease.Version.Bump.Minor; release with-defaults")
-addCommandAlias("releasePatch", ";set releaseVersionBump := sbtrelease.Version.Bump.Bugfix; release with-defaults")
\ No newline at end of file
+addCommandAlias("releaseNow", ";set releaseVersionBump := sbtrelease.Version.Bump.Bugfix; release with-defaults")
```
Comment (Zejnilovic) [inline build.sbt]: Add a new line ending please.

File: examples/spark-cobol-app/README.md [hunk 8]
```diff
@@ -13,34 +13,46 @@ Here is the list of Spark Application examples:
...
-**To run this on cluster generate the uber jar by running**
+**To build an uber jar to run on cluster**
 ```
 mvn package -DskipTests=true
 ```
-After the project is packaged you can copy 'target/spark-cobol-app-0.0.1-SNAPSHOT.jar'
+
+## Running via spark-submit
+
+After the project is packaged you can copy `target/spark-cobol-app-0.0.1-SNAPSHOT.jar` (if build using `Maven`) or `target/scala-2.11/spark-cobol-app-assembly-0.1.0-SNAPSHOT.jar` (if built using `sbt`)
```
Comment (Zejnilovic) [inline examples/spark-cobol-app/README.md]:
```suggestion
After the project is packaged you can copy `target/spark-cobol-app-0.0.1-SNAPSHOT.jar` (if built using `Maven`) or `target/scala-2.11/spark-cobol-app-assembly-0.1.0-SNAPSHOT.jar` (if built using `sbt`)
```

**Pull request merged**